cypress/integration/t2.js

```diff
@@ -0,0 +1,35 @@
+describe('User roles', function() {
+        
+    it('Should display validation for empty user after attempted loggin', function () {
+        cy.visit('/')
+        cy.get('.Select.not-valid').should('not.visible')
+        cy.get('[type="submit"]').click()
+        cy.get('.Select.not-valid').should('be.visible')
+    })
+
+    it('Should be able to login with role Test Lead', function () {
+        var roles = ["User", "Team Lead", "Manager", "Accountant", "Admin" ];        
+        var tabs = [1,2,5,5,6];        
+        roles.forEach(myFunction);
+
+        function myFunction(value, index, array) {
```

Comment:
Is myFunction the best name for this function? Maybe there is more specific name that you could name your function?